### DISCUSSION TITLE
Custom template for automation is not showing © {{ currentYear }} {{ company }}.

### DISCUSSION THREAD

**Discussion by zielono77** [Budibase/budibase#9472] [Help] [answered]
Hey,
Recently, I have created an app for business travels for company. At the moment, I encountered one problem with templates. The {{ currentYear }} {{ company }} is not showing in the emails once they have been sent. However, other emails such as password recovery, user welcome etc. works perfectly fine.
![Screenshot 2023-01-30 at 10 57 49](https://user-images.githubusercontent.com/24386954/215445696-bc7571bd-78a2-4ff5-a368-e712d0a2cff8.png)

Do you guys know where I'm making a mistake? :)

**Comment by melohagan** [comment, marked as answer]
Hey @zielono77 

Thanks for highlighting this. I've raised a bug here: https://github.com/Budibase/budibase/issues/9668
